llmops-article-classifier/README.md

```diff
@@ -0,0 +1,237 @@
+   - To access GPUs with sufficient VRAM, you may need to increase your GPU quota ([AWS](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html), [GCP](https://console.cloud.google.com/iam-admin/quotas), [Azure](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-quotas?view=azureml-api-2#request-quota-and-limit-increases))
+   - If the CUDA version on your GPU instance is incompatible with the default Docker image, modify it in the configuration file. See [available PyTorch images](https://hub.docker.com/r/pytorch/pytorch/tags)
+
+   ```shell
+   # Register a complete stack with GPU support
+   zenml stack register your-gcp-gpu-stack -o <ORCHESTRATOR_NAME> \
+       -a <ARTIFACT_STORE_NAME> \
+       -c <CONTAINER_REGISTRY_NAME> \
+       [-s <STEP_OPERATOR_NAME>]
+   ```
+
+   - Install the ZenML integration (`gcp`, `aws`, or `azure`):
+
+     ```shell
+     zenml integration install <INTEGRATION_NAME>
+     ```
+
+2. **Update the config file**:
+
+   - Modify the `remote_finetune.yaml` file to use your remote stack.
+   - You may also specify/modify the docker settings in the config file.
+
+3. **Launch remote training**:
+   ```shell
+   python run.py train --config remote_finetune.yaml
+   ```
+
+#### Additional training options:
+
+- `--config`: Path to YAML config file (defaults to "base_config.yaml")
+- `--save-test`: Save test set to disk for later evaluation
+- `--no-cache`: Disable caching for the pipeline run
+
+### Step 3: Evaluate and compare the fine-tuned ModernBERT model against Claude Haiku
+
+```bash
+python run.py compare
+```
+
+<p align="center">
+  <img src="assets/modernbert_claude_haiku_comparison.png" alt="Model Comparison" width="600"/>
+</p>
+
+### Step 4: Deploy your model (Optional)
+
+- Deploys the fine-tuned ModernBERT model to Hugging Face Hub
+- Requires a Hugging Face account with API token (+ write access)
+- Modify `huggingface_repo` in the `base_config.yaml` file with your own repo name
+
+```bash
+python run.py deploy
+```
+
+## 🛠️ Getting Started
+
+### Prerequisites
+
+- Python 3.9+
+- ZenML 0.75.0+
+- Hugging Face account with API token
+- Anthropic API key (optional, for model comparison)
+
+### Installation
+
+1. Clone the repository
+
+   ```bash
+   git clone https://github.com/zenml-io/llmops-article-classifier-demo
+   cd llmops-article-classifier-demo
+   ```
+
+2. Create and activate virtual environment
+
+   ```bash
+   python -m venv venv
+   source venv/bin/activate  # On Windows: venv\Scripts\activate
+   ```
+
+3. Install dependencies
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+4. Set up environment variables
+
+   ```bash
+   cp .env.example .env
+   # Edit .env file to add your HF_TOKEN and ANTHROPIC_API_KEY
+   ```
+
+5. Initialize ZenML (if not already done)
+   ```bash
+   zenml init
+   ```
```

Comment:
I think all these little blocks might need / want to be aligned with the far left margin otherwise the markdown parser seems to struggle?